ui/partials/main/thGlobalTopNavPanel.html

```diff
@@ -24,23 +24,25 @@
         <ul id="notification-dropdown" class="dropdown-menu" role="menu" aria-labelledby="notificationLabel">
           <li role="presentation" class="dropdown-header" title="Notifications">
             Recent notifications
-            <span class="label label-default" ng-show="notifications().length" ng-click="clear()">Clear all</span>
+            <button class="btn btn-xs btn-default notification-dropdown-btn" title="Clear all notifications"
+                  ng-show="notifications().length" ng-click="clear()">Clear all</button>
           </li>
           <li ng-show="!notifications().length">
-            <a class="text-muted">No recent notifications</a>
+            <span>No recent notifications</span>
```

Comment:
I debated about leaving the `text-muted` class in here, but plain text provided some contrast between it and the dropdown header. I can put it back in if we want.